.github/workflows/swift-toolchain.yml

```
@@ -343,11 +343,10 @@ jobs:
           echo "hash=$hash" >> $env:GITHUB_OUTPUT
 
       - name: Setup sccache
-        uses: hendrikmuhs/ccache-action@2e0e89e8d74340a03f75d58d02aae4c5ee1b15c6
+        uses: ./SourceCache/swift-build/.github/actions/setup-sccache
```

```suggestion
        uses: ${{ github.workspace }}/SourceCache/swift-build/.github/actions/setup-sccache
```

The same throughout please

---

I had tried this setup here https://github.com/thebrowsercompany/swift-build/pull/263/commits/7126feda12ef3e5a7ac1994ccb5e60ba1ae1ba1e and it didn't seem like it was valid by GitHub's standards (the workflow did not start when I was using ${{ github.workspace }}). Additionally, I don't see any mention of workspace expansion in the github docs for this https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses. I'd prefer to leave it as is for the time since I do know that it works.